From 1bc56aad2c40ab30c57c15536f9d09bb12f11087 Mon Sep 17 00:00:00 2001
From: Wim Taymans <wtaymans@redhat.com>
Date: Thu, 22 Feb 2018 11:37:07 +0100
Subject: [PATCH 2/8] video: disable ORC_RESTRICT

We pass the same pointers to the video_chroma_up,
video_chroma_down fnd video_convert unctions, which causes the
compiler to complain.
---
 gst-libs/gst/video/video-chroma.c    | 2 ++
 gst-libs/gst/video/video-converter.c | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/gst-libs/gst/video/video-chroma.c b/gst-libs/gst/video/video-chroma.c
index f761b4435..7ed22ea1e 100644
--- a/gst-libs/gst/video/video-chroma.c
+++ b/gst-libs/gst/video/video-chroma.c
@@ -24,6 +24,8 @@
 #include <string.h>
 #include <stdio.h>
 
+#define ORC_RESTRICT
+
 #include "video-orc.h"
 #include "video-format.h"
 
diff --git a/gst-libs/gst/video/video-converter.c b/gst-libs/gst/video/video-converter.c
index 22f81a465..147252415 100644
--- a/gst-libs/gst/video/video-converter.c
+++ b/gst-libs/gst/video/video-converter.c
@@ -29,6 +29,8 @@
 #endif
 #endif
 
+#define ORC_RESTRICT
+
 #include "video-converter.h"
 
 #include <glib.h>
-- 
2.47.0

